b288702/README.md

```diff
@@ -93,132 +95,123 @@ other. Because each verification step requires two sequential contribution files
 the same storage should be verified sequentially. Otherwise, you risk two adjacent verifications simultaneously
 requesting the same file.
 
-There is one additional dependency: `verify_initial.sh` must be run before `verify_contrib.sh` for the first contrib of
-every circuit. This is because the 'before' parameters for the first transition cannot be downloaded: they must be
-deterministically generated from the target circuit.
-
 ```bash
 bash -c '
   set -e
-  ./verify_initial.sh winning 32
-  ./verify_contrib.sh winning 32 1
-  ./verify_contrib.sh winning 32 2
-  ./verify_contrib.sh winning 32 3
-  ./verify_contrib.sh winning 32 4
-  ./verify_contrib.sh winning 32 5
-  ./verify_contrib.sh winning 32 6
-  ./verify_contrib.sh winning 32 7
-  ./verify_contrib.sh winning 32 8
-  ./verify_contrib.sh winning 32 9
-  ./verify_contrib.sh winning 32 10
-  ./verify_contrib.sh winning 32 11
-  ./verify_contrib.sh winning 32 12
-  ./verify_contrib.sh winning 32 13
-  ./verify_contrib.sh winning 32 14
-  ./verify_contrib.sh winning 32 15
-  ./verify_contrib.sh winning 32 16
-  ./verify_contrib.sh winning 32 17
-  ./verify_contrib.sh winning 32 18
-  ./verify_contrib.sh winning 32 19
-  ./verify_contrib.sh winning 32 20
+
+  ./download_initial_generation_prereqs.sh winning 32 && ./generate_initial.sh winning 32 && ./verify_contrib.sh winning 32 1
+  ./download_prereqs_contrib.sh winning 32 2 && ./verify_contrib.sh winning 32 2
+  ./download_prereqs_contrib.sh winning 32 3 && ./verify_contrib.sh winning 32 3
+  ./download_prereqs_contrib.sh winning 32 4 && ./verify_contrib.sh winning 32 4
+  ./download_prereqs_contrib.sh winning 32 5 && ./verify_contrib.sh winning 32 5
+  ./download_prereqs_contrib.sh winning 32 6 && ./verify_contrib.sh winning 32 6
+  ./download_prereqs_contrib.sh winning 32 7 && ./verify_contrib.sh winning 32 7
+  ./download_prereqs_contrib.sh winning 32 8 && ./verify_contrib.sh winning 32 8
+  ./download_prereqs_contrib.sh winning 32 9 && ./verify_contrib.sh winning 32 9
+  ./download_prereqs_contrib.sh winning 32 10 && ./verify_contrib.sh winning 32 10
+  ./download_prereqs_contrib.sh winning 32 11 && ./verify_contrib.sh winning 32 11
+  ./download_prereqs_contrib.sh winning 32 12 && ./verify_contrib.sh winning 32 12
+  ./download_prereqs_contrib.sh winning 32 13 && ./verify_contrib.sh winning 32 13
+  ./download_prereqs_contrib.sh winning 32 14 && ./verify_contrib.sh winning 32 14
+  ./download_prereqs_contrib.sh winning 32 15 && ./verify_contrib.sh winning 32 15
+  ./download_prereqs_contrib.sh winning 32 16 && ./verify_contrib.sh winning 32 16
+  ./download_prereqs_contrib.sh winning 32 17 && ./verify_contrib.sh winning 32 17
+  ./download_prereqs_contrib.sh winning 32 18 && ./verify_contrib.sh winning 32 18
+  ./download_prereqs_contrib.sh winning 32 19 && ./verify_contrib.sh winning 32 19
+  ./download_prereqs_contrib.sh winning 32 20 && ./verify_contrib.sh winning 32 20
```

Comment:
( could not resist )

![Charlie-Sheen-Winning-Poster](https://user-images.githubusercontent.com/356110/90676779-c64c9e00-e25c-11ea-877c-4483217c877d.jpg)